res/values-pt-rBR/strings.xml

```diff
@@ -976,13 +976,13 @@ Se você notar um jogador quebrando as regras do jogo, denuncie a um Moderador.
     <string name="OPEN">ABRIR</string>
     <string name="Halo">Aréola</string>
     <string name="To_use_this_halo_you_must_be_at_least_level_s_and_unlock_s_achievements">"Para usar esta aréola, você deve ter pelo menos o nível  %1$s e ganhar %2$s conquistas."</string>
-    <string name="Halos">Aréolas</string>
+    <string name="Halos">Auréolas</string>
```

Comment:
Both "aréolas" and "auréolas" are already correct. Do not change it, as people are already used to "aréolas". You can even make a research of it for proof.